packages/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts

```diff
@@ -77,8 +82,8 @@ See \`https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshoo
   }
 }
 
-export class NativeReanimated {
-  private InnerNativeModule: NativeReanimatedModule;
+class NativeReanimatedModule implements IReanimatedModule {
+  #reanimatedModuleProxy: ReanimatedModuleProxy;
```

Comment:
I guess we should also rename the C++ counterpart (`NativeReanimatedModule` to `ReanimatedModuleProxy`) but we can do it in a separate PR.

Reply:
AFAIK we can't because of the Codegen. I wanted to have `WorkletsCommonModule` in the first place but I was forced to change the name to `WorkletsNativeModule` - `NativeModule` suffix is required.